src/bin/bench-tps.rs

```diff
@@ -543,7 +550,20 @@ fn main() {
     let barrier_id = rnd.gen_n_keypairs(1).pop().unwrap();
 
     println!("Get tokens...");
-    airdrop_tokens(&mut client, &leader, &id, tx_count);
+    let num_tokens_per_account = 20;
+
+    // Sample the first keypair, see if it has tokens, if so then resume
+    // to avoid token loss
+    let keypair0_balance = client.poll_get_balance(&keypairs[0].pubkey()).unwrap_or(0);
+
+    if num_tokens_per_account > keypair0_balance {
```

Comment:
Won't this always evaluate to true?

Reply:
looks right to me. It's asking, "Do I need more tokens than I have? If so, airdrop."  Might be easier to grok if it were flipped around `keypair0_balance < num_tokens_per_account`: "Do I have fewer tokens than I need? If so, airdrop."

Reply:
If `should_switch_directions` is working and `i = keypair0_balance`, keypair0 won't ever receive more than 3/4 of `num_tokens_per_account` and so will always be less...

Reply:
It's for the case that the user modifies this value for a subsequent run. Imagine a user uses `num_tokens_per_account=100` initially then changes it to `=10`. Then the balance could be greater.

Reply:
Ah, fair point. Okay, I dig.